### PR TITLE
enh (real-time): Real-time SQL queries modified to distinguish them from configuration SQL queries

### PR DESCRIPTION
**Pull request by callapa**
## Description

The aim is to modify the queries for the real time and their SELECT FOUND_ROWS() queries so that they can be operated differently from other configuration queries (e.g. proxySQL, ...).

## Type of change

- [ ] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [x] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [x] 23.04.x
- [x] 23.10.x (master)

#### Community contributors & Centreon team

- [x] I have followed the **coding style guidelines** provided by Centreon
- [x] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [x] I have commented my code, especially **hard-to-understand areas** of the PR.
- [x] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by dmyios — DISMISSED** [review]

**Review by adr-mo — APPROVED** [review]

**Review by dmyios — DISMISSED** [review]

**Review by adr-mo — APPROVED** [review]

File: centreon/src/Core/Infrastructure/RealTime/Repository/Icon/DbReadHostgroupRepository.php [hunk 1]
```diff
@@ -21,12 +21,13 @@
 
 declare(strict_types=1);
 
-namespace Core\Infrastructure\RealTime\Repository\Hostgroup;
+namespace Core\Infrastructure\RealTime\Repository\Icon;
```
Comment (a-launois) [inline centreon/src/Core/Infrastructure/RealTime/Repository/Icon/DbReadHostgroupRepository.php:24]: Why this change of namespace ?

Comment (callapa) [reply centreon/src/Core/Infrastructure/RealTime/Repository/Icon/DbReadHostgroupRepository.php:24]: Oops, a handling error

**Review by a-launois — APPROVED** [review]

**Review by dmyios — APPROVED** [review]
